index.js

```diff
@@ -297,6 +297,14 @@ function prepareRequest(options) {
       case 'tempDir': {
         break;
       }
+      case 'overwrite': {
```

Comment:
Does this only apply to `overwrite`, and not any of the other boolean flags?

Reply:
Only `overwrite` as that's the only one that gets sent to the API. All of the other boolean args configure the behaviour of this module but don't get put in the payload.